apps/studio/src/components/PageEditor/ComponentSelector.tsx

```diff
@@ -29,6 +29,7 @@ import {
 
```

Comment:
Some qns on UX - shouldn't we be showing a toast when save is successful and when it is not?

Reply:
this doesn't save

Reply:
> Only if they pass this check and have valid bounds for the from, to tuple, we save the updated state into our db.

It's mentioned you save the state in db?

Reply:
just selecting the component itself doesn't do anything, we require them to drag/drop in order to display feedback! that part is done via a toast in the mutation itself in `rootstatedrawer`!

Reply:
Hmm I am referring to your demo video where you are doing drag and drops of component blocks. Shouldn't each drag and drop show a toast saying the action has saved successfully?